planner/core/rule_inject_extra_projection.go

```diff
@@ -20,11 +20,11 @@ import (
 	"github.com/pingcap/tidb/sessionctx"
 )
 
-// injectExtraProjection is used to extract the expressions of specific
+// InjectExtraProjection is used to extract the expressions of specific
 // operators into a physical Projection operator and inject the Projection below
 // the operators. Thus we can accelerate the expression evaluation by eager
 // evaluation.
-func injectExtraProjection(plan PhysicalPlan) PhysicalPlan {
+func InjectExtraProjection(plan PhysicalPlan) PhysicalPlan {
```

Comment:
It is better to add some comments to explain the function is to change the cop plan for `cop[tidb]`.